docs/releases/2_0.rst

```diff
@@ -0,0 +1,83 @@
+Version 2.0
+===========
+
+Version 2.0.0
+-------------
+
+This version adds the following features, outlined below. Please note that
+pVACtools 2.0 is not backwards-compatible and certain changes will break old
+workflows.
+
+Breaking changes
+________________
+
+- pVACtools now supports variable epitope lengths for class II prediction algorithms. The previous option
+  ``--epitope-length`` (``-e``) no longer exists. It has been replaced with
+  ``--class-i-epitope-length`` (``-e1``) and ``--class-ii-epitope-length``
+  (``-e2``) for class I and class II epitope lengths, respectively. The
+  defaults are ``[8, 9, 10, 11]`` and ``[12, 13, 14, 15, 16, 17, 18]``,
+  respectively.
+- The ``--peptide-sequence-length`` option has been removed. The peptide
+  sequence length is now determined by the epitope length(s) to determine the
+  flanking sequence length before and after the mutation.
+- pVACtools no longer depends on conda. pVACtools remains compatible with
+  Python 3.5 and above but users may chose any environment manager to set up
+  an appropriate Python environment.
+- When using standalone IEDB, pVACtools is now only compatible with IEDB 3.1
+  and above. Please see :ref:`install` for instructions on installing the
+  latest IEDB version.
+- pVACseq is no longer dependent on annotations with the VEP Downstream
+  plugin. This dependency has been replaced with the VEP Frameshift plugin.
+  This requires changes to your existing VEP installation in order to install
+  the Frameshift plugin. Existing VCFs that were previously annotated to work
+  with pVACtools 1.5 and below will no longer work with version 2.0 and above
+  and will need to be reannotated. Please see our documentation on :ref:`vep`
+  for more information.
+- The filtered.condensed.tsv report has been removed and replaced with the
+  all_epitopes.aggregated.tsv report. We believe that this new report will
+  provide a more comprehensive and easier to understand summary of your
+  results. Please see the Output Files sections of each tool for more
+  information on this new report.
+
+New features
+____________
+
+- pVACtools now provides ic50 binding affinity percentile rank information, in
+  addition to the raw ic50 binding affinity values. Users may filter on the
+  percentile rank by using the new ``--percentile-threshold`` argument.
+- Users now have the option of calculating the reference proteome similarity
+  of their filtered epitopes. For this, the peptide sequence for the
+  remaining variants is mapped to the reference proteome using BLAST. Variants
```

Comment:
Sorry if I'm missing this but I couldn't find the --percentile-threshold and --run-reference-proteome-similarity on the pvacseq usage page as one of the options?

Reply:
Those will be auto-populated in the usage docs once the new version is actually released.